src/panels/config/automation/add-automation-element-dialog.ts

```diff
@@ -363,14 +363,14 @@ class DialogAddAutomationElement extends LitElement implements HassDialog {
               ></ha-service-icon>
             `,
             key: `${SERVICE_PREFIX}${dmn}.${service}`,
-            name: `${domain ? "" : `${domainToName(localize, dmn)}: `}${
+            name: `${dmn ? "" : `${domainToName(localize, dmn)}: `}${
```

Comment:
Why do we set it to an empty string when domain is not falsy?

Reply:
Yeah, adjusted.